addons/account/models/account_account.py

```diff
@@ -738,12 +738,12 @@ def _set_opening_debit_credit(self, amount, field):
         data[self.env.company.id][self.id][index] = amount
 
     @api.model
-    def default_get(self, default_fields):
+    def default_get(self, fields):
```

Comment:
clashes with "from odoo import fields"

Reply:
I adapted all code where there was a clash.
I aligned the parameter name to what is available in other methods like `search_read`, before this there is a nearly 50/50 distribution between `fields` and `field_list` for this method.